akka-docs-new/src/main/paradox/general/stream/stream-design.md

```diff
@@ -0,0 +1,106 @@
+<a id="stream-design"></a>
+# Design Principles behind Akka Streams
+
+It took quite a while until we were reasonably happy with the look and feel of the API and the architecture of the implementation, and while being guided by intuition the design phase was very much exploratory research. This section details the findings and codifies them into a set of principles that have emerged during the process.
+
+@@@ note
+
+As detailed in the introduction keep in mind that the Akka Streams API is completely decoupled from the Reactive Streams interfaces which are just an implementation detail for how to pass stream data between individual processing stages.
+
+@@@
+
+## What shall users of Akka Streams expect?
+
+Akka is built upon a conscious decision to offer APIs that are minimal and consistent—as opposed to easy or intuitive. The credo is that we favor explicitness over magic, and if we provide a feature then it must work always, no exceptions. Another way to say this is that we minimize the number of rules a user has to learn instead of trying to keep the rules close to what we think users might expect.
+
+From this follows that the principles implemented by Akka Streams are:
+
+>
+ * all features are explicit in the API, no magic
+ * supreme compositionality: combined pieces retain the function of each part
+ * exhaustive model of the domain of distributed bounded stream processing
+
+This means that we provide all the tools necessary to express any stream processing topology, that we model all the essential aspects of this domain (back-pressure, buffering, transformations, failure recovery, etc.) and that whatever the user builds is reusable in a larger context.
+
+### Akka Streams does not send dropped stream elements to the dead letter office
+
+One important consequence of offering only features that can be relied upon is the restriction that Akka Streams cannot ensure that all objects sent through a processing topology will be processed. Elements can be dropped for a number of reasons:
+
+>
```

Comment:
it seems this causes the bulleted list to be indented as well as bulleted. Not sure if that makes sense to me, but it's consistent between the RST and Paradox versions